### PR TITLE
Migrate SpeedSlider storybook entry

### PR DESCRIPTION
**Pull request by fisher-alice**
This PR migrates the `SpeedSlider` component to the updated Storybook API.

Video of component in Storybook:

https://github.com/code-dot-org/code-dot-org/assets/107423305/bf81bb64-d192-4aa7-b208-ea42eb24098d

## Links
[jira](https://codedotorg.atlassian.net/browse/LABS-477)

## Follow-up work
I noticed in `SpeedSlider.jsx` that the [dark turtle and rabbit images](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/SpeedSlider.jsx#L24-L27) are not used. Currently, this component is used in the Debug Console area in App Lab and in the Game Lab animation tab if an animation with more than one frame is added. For both uses, the background is dark so the light color UI is appropriate.

Screenshot 2024-02-01 at 11 11 02 AM

Screenshot 2024-02-01 at 11 11 21 AM

But I wonder if we should update the `SpeedSlider` component so it has both white and dark UI available (similar to how `color` is implemented in [`SimpleDropdown.tsx`](https://github.com/code-dot-org/code-dot-org/blob/f109907bbc4d23238b184548a56f8a3b33da97c1/apps/src/componentLibrary/simpleDropdown/SimpleDropdown.tsx#L29)). 

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/SpeedSlider.story.jsx [hunk 1]
```diff
@@ -8,17 +8,25 @@ class StorybookHarness extends React.Component {
...
     );
   }
 }
 
-export default storybook => {
-  return storybook
-    .storiesOf('SpeedSlider', module)
-    .add('Default', () => <StorybookHarness />);
+export default {
+  title: 'SpeedSlider',
```
Comment (bencodeorg) [inline apps/src/templates/SpeedSlider.story.jsx]: Assuming we get no objections to storybook reorganization, remove this?

**Review by bencodeorg — APPROVED** [review]
LGTM -- curious if the harness effect could be done without react/state, ie:

```
let value = 0.5;
const onValueChange = newValue => value = newValue;
```

But fine as-is

**Comment by fisher-alice** [comment]
> LGTM -- curious if the harness effect could be done without react/state, ie:
> 
> ```
> let value = 0.5;
> const onValueChange = newValue => value = newValue;
> ```
> 
> But fine as-is

Thanks! I tried your suggestion, but then the slider was not movable. So will leave as is.

Just noting that same harness approach is used in `PaginationWrapper.story.jsx`.

**Review by levadadenys — APPROVED** [review]
LGTM!

**Comment by levadadenys** [comment]
> But I wonder if we should update the `SpeedSlider` component so it has both white and dark UI available (similar to how `color` is implemented in [`SimpleDropdown.tsx`](https://github.com/code-dot-org/code-dot-org/blob/f109907bbc4d23238b184548a56f8a3b33da97c1/apps/src/componentLibrary/simpleDropdown/SimpleDropdown.tsx#L29)).

We have `SpeedSlider` component implementatin called `Slider` in scope of Design System (in Figma only for now), designs of it are not final, but right now it has only light icons color, the only case where icons are slightly darker - disabled state of the `Slider`. 

So I guess for now we should be fine with what we have in SpeedSlider, no need/rush to implement dark styles right now.

**Comment by fisher-alice** [comment]
> > But I wonder if we should update the `SpeedSlider` component so it has both white and dark UI available (similar to how `color` is implemented in [`SimpleDropdown.tsx`](https://github.com/code-dot-org/code-dot-org/blob/f109907bbc4d23238b184548a56f8a3b33da97c1/apps/src/componentLibrary/simpleDropdown/SimpleDropdown.tsx#L29)).
> 
> We have `SpeedSlider` component implementatin called `Slider` in scope of Design System (in Figma only for now), designs of it are not final, but right now it has only light icons color, the only case where icons are slightly darker - disabled state of the `Slider`.
> 
> So I guess for now we should be fine with what we have in SpeedSlider, no need/rush to implement dark styles right now.

Awesome - thanks for the update @levadadenys !

**Pull request merged**